python/tk_desktop2/websockets/util.py

```diff
@@ -68,12 +66,14 @@ def _convert(data):
     :returns: Object with only utf-8 encoded strings
     """
 
+    if isinstance(data, six.string_types):
```

Comment:
A string is iterable and was looping infinitely. since it was constantly calling _convert on the first character.